src/utils.js

```diff
@@ -29,7 +29,6 @@ const mimeType = async (opts) => {
   const { path, buffer } = opts;
   const magicFile = PATH.join(__dirname, '../assets/magic.mgc');
   if (fs.existsSync(magicFile)) FileMagic.magicFile = magicFile;
-  FileMagic.defaulFlags = MagicFlags.MAGIC_PRESERVE_ATIME;
```

Comment:
Yes, it's affecting this pull request because of https://github.com/paulmillr/chokidar/issues/1224, as explained on https://github.com/iterative/cml/pull/1026#issue-1250268830

Reply:
> Yes, it's affecting this pull request because of [paulmillr/chokidar#1224](https://github.com/paulmillr/chokidar/issues/1224), as explained on [#1026 (comment)](https://github.com/iterative/cml/pull/1026#issue-1250268830)

Where exactly? I can not find that comment

Reply:
![Captura de pantalla 2022-06-09 a las 16 29 16](https://user-images.githubusercontent.com/11387611/172871837-1d953fc0-e547-4598-a2b1-23cc657b9e18.png)

Reply:
It's not the case, unless proven otherwise

Reply:
Errrr OK